package.json

```diff
@@ -95,6 +95,7 @@
   },
   "dependencies": {
     "babel-runtime": "6.9.2",
-    "vue": "1.0.24"
+    "vue": "1.0.24",
+    "vue-multiselect": "^1.0.1"
```

Comment:
How do I do that ? I have forked the project, but I don't know how to update my fork.
Also, I can't seem to make `vue-multiselect` a optional dependency without breaking everything.
I'm currently using `components` to make `multiselect` available to the template, but I don't know how to load it optionally. I tried with a function that return either an empty object or the component definition from the lib (VueMultiselect.Multiselect). It fail badly.
I can't really do like `bootstrap-select` or `ionRangeSlider` because vue-multiselect is really tied to Vue.
Help ?